physics/MP/Thompson/mp_thompson.meta

```diff
@@ -282,6 +282,12 @@
   kind = kind_phys
   intent = in
   optional = True
+[is_initialized]
+  standard_name = flag_for_thompson_mp_scheme_initialization
+  long_name = flag carrying scheme initialization status
+  units = flag
```

Comment:
Same thing here. Missing dimensions causing ccpp_prebuild to fail.

Reply:
Sorry, I had un committed changes. My bad.